ansible_collections/arista/avd/molecule/eos_designs-mpls-isis-sr-ldp/inventory/group_vars/MPLS_TENANTS_NETWORKS.yml

```diff
@@ -97,7 +97,7 @@ tenants:
           2020:
             enabled: true
             name: TENANT_B_INSIDE_FW
-            ip_virtual_router_address: 10.1.255.1
+            # ip_virtual_router_address: 10.1.255.1
```

Comment:
Let's remove this if we don't want it, or change to the list form, if we want it.